### ISSUE TITLE
save query & continue to edit

### ISSUE THREAD

**Issue by wellsoliver** [jakob/Postico#106]
(apologies if this is a dupe) I'd love it if the Save Query button then put Postico into an edit mode, with that file loaded, and continued saves just wrote to the same file. Also, it seems like it would be intuitive for command-S to fire off the save.

**Comment by jakob** [comment]
I agree that the current behavior is pretty annoying. I'm actually planning to change the way queries work; I want to allow using multiple query files, similar to the suggestion in issue #18.

Once I've implemented the new interface, I'm considering to just auto-save query files. Requiring explicit save commands is so 90ies...

**Comment by eoglethorpe** [comment]
+1 on easier query saving

**Comment by mritchie712** [comment]
+1

Any movement on this?

**Comment by jakob** [comment]
I've been working on a new sidebar with a file browser, that lets you use multiple queries and save them etc. It should solve this issue. It is still a very early prototype, but you can have a look here:

https://github.com/jakob/Postico/issues/18

**Comment by mritchie712** [comment]
Thanks! That's what I was looking for.

**Comment by goldcaddy77** [comment]
Hey, any updates on this.  Just bought Postico yesterday and loving it, but I'm trying to maintain a library of scripts that I can use and cmd+S not saving the current file is slowing things down quite a bit.

**Comment by kdelrosso** [comment]
+1

**Comment by tgig** [comment]
+1. Every time I open an existing query in a new tab, I have to double check that I'm saving over the right sql file when I save progress. I like the concept in #18, and would like to see it live. Decided to comment on this thread instead as it seems to have more action.

**Comment by mrkcohen** [comment]
+1

**Comment by jasonl99** [comment]
+ 1,000 (I've lost work as a result of the horribly non-intuitive way of saving the current sql file I'm working on).
